exporter/bmchelixexporter/metrics_client.go

```diff
@@ -0,0 +1,94 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package bmchelixexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/bmchelixexporter"
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/configopaque"
+	"go.uber.org/zap"
+)
+
+// MetricsClient is responsible for sending the metrics payload to BMC Helix Operations Management
+type MetricsClient struct {
```

Comment:
Just being mindful of what is being exported as part of the package, I think it just makes sense to rename as `metricsClient` and it should be fine here.

Reply:
Thanks! I will rename the struct name to `metricsClient`